activerecord/lib/active_record/counter_cache.rb

```diff
@@ -112,14 +155,36 @@ def increment_counter(counter_name, id)
       #
       # * +counter_name+ - The name of the field that should be decremented.
       # * +id+ - The id of the object that should be decremented or an array of ids.
+      # * +options+ - {touch: true} will update the `timestamp_attributes_for_update_in_model` column.
+      #               {touch: :last_discussed_at} will update the `last_discussed_at` column.
+      #               {touch: %i( updated_at last_discussed_at )} will update the `updated_at` and `last_discussed_at` columns.
```

Comment:
When we switch to a keyword arg we can just say:

```ruby
# * <tt>:touch</tt> - Touch timestamp columns when updating.
#   Pass +true+ to touch +updated_at+ and/or +updated_on+. Pass a symbol to
#   touch that column or an array of symbols to touch just those ones.
```

Let's an example or two down at the examples section. Then we'd need to adapt that to the other cases as well.